src/main/java/org/cryptomator/common/mount/Mounter.java

```diff
@@ -27,26 +28,31 @@
 @Singleton
 public class Mounter {
 
-	private static final List<String> CONFLICTING_MOUNT_SERVICES = List.of("org.cryptomator.frontend.fuse.mount.MacFuseMountProvider", "org.cryptomator.frontend.fuse.mount.FuseTMountProvider");
+	// mount providers (key) can not be used if any of the conflicting mount providers (values) are already in use
+	private static final Map<String, Set<String>> CONFLICTING_MOUNT_SERVICES = Map.of(
+			"org.cryptomator.frontend.fuse.mount.MacFuseMountProvider", Set.of("org.cryptomator.frontend.fuse.mount.FuseTMountProvider"),
+			"org.cryptomator.frontend.fuse.mount.FuseTMountProvider", Set.of("org.cryptomator.frontend.fuse.mount.MacFuseMountProvider")
```

Comment:
When using the mount provider name as the key, it becomes impossible to expand the map once another MountService conflicts with macFUSE or FUSE-T.

And how about encapsulating the mount provider in static variables.
`org.cryptomator.frontend.fuse.mount.MacFuseMountProvider` is also utilized in Settings.java

Reply:
Usually I support deduplication, but for class names I see some benefits in directly reading the value in source code.